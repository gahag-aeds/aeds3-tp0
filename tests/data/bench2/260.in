260 130
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 
Sub 80 165
Max 89 174
Add 127 212
Sum 84 169
Sub 64 149
Sum 144 229
Sub 10 95
Sum 144 229
Add 11 96
Max 56 141
Add 1 86
Max 173 258
Sub 146 231
Sum 93 178
Sub 156 241
Min 43 128
Add 43 128
Max 17 102
Sub 150 235
Min 1 86
Sub 24 109
Sum 168 253
Sub 106 191
Min 127 212
Add 65 150
Max 86 171
Sub 60 145
Min 13 98
Sub 122 207
Min 49 134
Sub 108 193
Min 19 104
Add 137 222
Sum 60 145
Sub 102 187
Sum 18 103
Add 29 114
Min 103 188
Add 161 246
Min 31 116
Sub 158 243
Sum 162 247
Add 29 114
Sum 129 214
Sub 72 157
Max 11 96
Add 163 248
Min 115 200
Add 19 104
Max 131 216
Add 115 200
Sum 42 127
Sub 124 209
Max 38 123
Sub 160 245
Max 14 99
Sub 124 209
Min 46 131
Add 19 104
Sum 63 148
Sub 94 179
Sum 126 211
Add 81 166
Sum 48 133
Add 3 88
Sum 174 259
Add 65 150
Sum 24 109
Sub 102 187
Min 43 128
Sub 54 139
Min 85 170
Sub 22 107
Max 83 168
Add 31 116
Min 94 179
Add 85 170
Min 19 104
Sub 26 111
Max 95 180
Add 149 234
Sum 132 217
Add 137 222
Min 91 176
Sub 170 255
Sum 114 199
Sub 96 181
Max 119 204
Add 159 244
Min 106 191
Add 173 258
Min 70 155
Add 57 142
Min 79 164
Add 109 194
Min 52 137
Sub 78 163
Sum 165 250
Add 75 160
Max 5 90
Sub 34 119
Max 128 213
Add 81 166
Min 55 140
Sub 28 113
Sum 111 196
Sub 140 225
Max 104 189
Add 121 206
Sum 165 250
Add 25 110
Min 88 173
Add 115 200
Sum 153 238
Sub 4 89
Max 110 195
Sub 92 177
Sum 99 184
Sub 46 131
Max 77 162
Add 31 116
Max 44 129
Sub 138 223
Min 79 164
Sub 114 199
Min 73 158
Sub 122 207
Max 17 102
Add 63 148
Min 22 107
