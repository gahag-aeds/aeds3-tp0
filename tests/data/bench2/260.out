99
4141
4471
4471
98
98
4099
-2
100
1
4501
-3
96
-2
-2
-2
3827
4185
-5
-1
4406
4063
100
-6
95
4115
101
101
-1
3901
3894
4162
4493
4553
-1
-8
95
-8
4
96
4037
-5
4000
93
-7
-2
-6
0
4638
103
95
1
4039
95
4621
-4
4382
97
3799
100
101
-5
-6
101
3
